trade-finance-manager-ui/server/routes/utilisation-reports/index.js

```diff
@@ -8,6 +8,8 @@ const router = express.Router();
 
 router.get('/', validateUserTeam(Object.values(PDC_TEAM_IDS)), getUtilisationReports);
 
+router.post('/', validateUserTeam([PDC_TEAM_IDS.PDC_RECONCILE]), updateUtilisationReportStatus);
```

Comment:
This needs to be `post` and not `put` due to the html form element not supporting `method="PUT"`